install.sh

```diff
@@ -10,7 +10,14 @@ glibc_filename="glibc-2.31.tar.xz"
 args=("$@")
 
 function build() {
-    (cd "./${prefix}-$1" && makepkg -i "${args[@]}")
+    (
+        cd "./${prefix}-$1"
+        package_filename=$(makepkg --packagelist)
+        if [[ ! -f "$package_filename" ]] ; then
```

Comment:
I sometimes do things like `./install.sh -Csf` in order to test that everything works end-to-end.  This breaks that workflow because it will never rebuild a package that's already built.

Not sure if there's a nice way to fix this while keeping it unattended.  You could check for `$? == 13` and stop on other errors.